### PR TITLE
Show node version on ares-device -i

### PR DESCRIPTION
**Pull request by YoungeunKim**
:Release Notes:
Show node version of platform on ares-device -i

:Detailed Notes:
Get node version information from systemservice and print it

:Testing Performed:
1. Passed CLI unit test on OSE target
2. Pass eslint
3. Verified with ares-device -i command. The result should have "nodejs_versions" 
  $ ares-device -i 
   > nodejs_versions : v12.21.0

:Issues Addressed:
[WRN-12204] Support node version information in ares-device

### CODE REVIEW

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

**Review by ssuminahn — COMMENTED** [review]
code :-1: 
verify :+1:

File: lib/device.js [hunk 1]
```diff
@@ -155,6 +156,31 @@ const async = require('async'),
...
 
+            function _getSoftwareInfo(next) {
+                log.info("device#systemInfo#_getSoftwareInfo()");
+                const target = options.session.getDevice(),
+                    addr = target.lunaAddr.softwareInfo,
+                    param = {
+                        parameters: ["nodejs_versions"]
+                    };
+
+                    luna.sendWithoutErrorHandle(options, addr, param, function(lineObj, next) {
```
[the remark this answers is not in the document]
Comment (YoungeunKim) [reply lib/device.js]: Fixed

**Comment by YoungeunKim** [comment]
Code 👍 
Verified 👍

**Review by ssuminahn — APPROVED** [review]
code :+1: 
verify :+1: 

I updated some of the commit message.

**Pull request merged**